Mathlib/RingTheory/PowerSeries/Inverse.lean

```diff
@@ -367,6 +367,14 @@ instance : NormalizationMonoid k⟦X⟧ where
     rw [inv_inj, Units.ext_iff, ← hu, Unit_of_divided_by_X_pow_order_nonzero h₀.ne_zero]
     exact ((eq_divided_by_X_pow_order_Iff_Unit h₀.ne_zero).mpr h₀).symm
 
+theorem normUnit_X : normUnit (PowerSeries.X : PowerSeries k) = 1 := by
+  dsimp only [normUnit]
+  rw [inv_eq_one, ← Units.val_eq_one, Unit_of_divided_by_X_pow_order_nonzero,
+    divided_by_X_pow_order_of_X_eq_one]
```

Comment:
```suggestion
theorem normUnit_X : normUnit (X : PowerSeries k) = 1 := by
  simp [normUnit, ← Units.val_eq_one, Unit_of_divided_by_X_pow_order_nonzero]
```
Also, it can be `simp`.